README.md

```diff
@@ -3,9 +3,33 @@
 This framework is not yet stable and is subject to change, but is available
 for early testing.
 
-## Getting Started
+# Getting Started
 
-Your charm directory should have the following overall structure:
+## Table of contents
+
+[Introduction](##Introduction)
+
+[Charm Creation](##Charm%20Creation)
+
+## Introduction
+
+### Juju and Charms
+
+The Charm Operator Framework is the companion to [Juju](https://jaas.ai/docs/what-is-juju) charms are the workers, and juju is the orchestrator, together they encapsulate the complexity of Day zero through to day three operations away from the user making infrastructure as a code a reality.
```

Comment:
We don't need to sell people on Juju here, I don't think. If they're looking at the framework, they already know Juju and its benefits.

```suggestion
The Charm Operator Framework is the companion to [Juju](https://jaas.ai/docs/what-is-juju). Charms are operators, software that runs software. Juju is the orchestrator, the tool that manages operators. The framework greatly simplifies writing and maintaining charms.
```

Reply:
Is that a safe assuption though? When we're talking about the operator framework with customers we're talking about the operator framework, sometimes we're simply mentioning Juju whilst focusing on the benefits of the Framework. Does it really hurt to give people more info? I personally had no idea about the relationship between the two and found it difficult to find this in the docs.

Reply:
As a principle, a usage guide should be a usage guide. People reading these docs want to know about how to write a charm.

I am a little uneasy about the idea of the framework being the entry point into the community.. but I guess I've misinterpreted the intention here?